### ISSUE TITLE
add likelihood components to sensitivity table 

### ISSUE THREAD

**Issue by aaronmberger** [cgrandin/hake-assessment#429]
Sampson suggested including the likelihood components in the sensitivity tables (Table 32 and 33 in 2018 assessment)... tagging this for 2019 go around.

**Comment by taylori** [comment]
From two seats away, the table looks good so far. The one thing that the SRG might want is to separate the contributions to the age comp likelihood for the fishery and the survey.

The list created r4ss has a `$likelihoods_by_fleet` component in addition to `$likelihoods_used` that would have that breakdown. The formatting of those tables isn't great but it should be easy to grab the necessary values.

**Comment by cgrandin** [comment]
Should I delete the zero value rows?

**Comment by taylori** [comment]
Right. I meant to chime in on that. Yes, I think any row that is either zero or the same for all models could probably be deleted.
The ones I would expect to change are
- Total
- Survey
- Age_comp (Fishery)
- Age_comp (Survey)
- Parm_priors (mostly from steepness and M)
- Recruitment (penalty on recdevs away from 0), and
- Parm_devs (penalty on time-varying recruitment)

Should be able to delete
- Equil_catch
- Forecast_Recruitment
- Crash_Pen

**Comment by andrew-edwards** [comment]
Maybe delete the zero rows (unless it's too fiddly to keep automated), but keep the ones that are the same for all models should stay (else the log-likelihoods won't add up).

**Comment by cgrandin** [comment]
Done in commit bc8570b58ab63c8bbfefa43e7ca6747b37adac31

I'm going to fix the SRG Requests beamer now and add the table in there.

**Comment by andrew-edwards** [comment]
Think this got done (despite being tagged for 2019). We can take another less-rushed look at it next year, just to confirm.

**Comment by kellijohnson** [comment]
I checked the likelihood values for the base model and one sensitivity against the report files and the correct values are used for the table. I think we can close this, but I will let someone else who did the work and knows more what the SRG was asking for close the issue.
